bicep/deploy/main.bicep

```diff
@@ -0,0 +1,53 @@
+targetScope = 'subscription'
+
+@allowed([
+  'Dev'
+  'Test'
+  'Prod'
+])
+@maxLength(4)
+@description('The environment name')
+param environmentName string = 'Dev'
+
+@allowed([
+  'DevTest'
+  'Production'
+])
+@maxLength(10)
+@description('The environment type')
+param environmentType string = 'DevTest'
+
+@allowed([
+  'westeurope'
+])
+@description('Location of resource group and resources')
+param location string
+
+// Configurations
+var constants = loadJsonContent('config-constants.json')
+var tags = loadJsonContent('config-tags.json')['${environmentType}']
+
+// Naming
+var resourceGroupName = '${constants.SystemName}-${toUpper(environmentName)}'
+var resourceName = '${constants.CompanyAbbreviation}${constants.SystemAbbreviation}${toLower(environmentName)}'
```

Comment:
Is it possible to apply `toLower()` on the entire `resourceName`?

Reply:
I'm a bit against removing dashes if it was explicitly specified from the outside

Reply:
I changed the whole resourcename to be toLower now

Reply:
> I'm a bit against removing dashes if it was explicitly specified from the outside

i agree with this